client/src/helpers/APICalls/getConversations.ts

```diff
@@ -0,0 +1,18 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const getConversations = async () => {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
+    credentials: 'include',
+  };
+  return await fetch(`/conversation/all`, fetchOptions)
+    .then((res) => res.json())
+    .catch((err) => {
+      err: {
+        message: err;
+      }
+    });
+};
```

Comment:
This should be:
`
.catch((err) => ({
      err: {
        message: err;
      }
    }));
`